internal/detector/jar_assessor.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+var Regex = *regexp.MustCompile("(.*)=(.*)")
```

Comment:
* Does this var need to be accessed outside the pkg ? Should be lower case 
* You souldn't need to dereference the result

Reply:
done